app/src/test/java/org/simple/clinic/analytics/AnalyticsTest.kt

```diff
@@ -31,13 +35,18 @@ class AnalyticsTest {
   }
 
   @Test
-  fun `when setting the user id without any reporters, no error should be thrown`() {
-    Analytics.setUserId(UUID.randomUUID())
+  fun `when setting the logged in user without any reporters, no error should be thrown`() {
+    Analytics.setLoggedInUser(user)
```

Comment:
Can we do something like,

```
try {
  Analytics.setLoggedInUser(user)
} catch (exception: Exception) {
  fail("Should not throw an exception: $exception")
}
```

Just the function invocation looks a bit suspicious.

Reply:
Good point, I'll do this.